docs/src/main/asciidoc/feature-management.adoc

```diff
@@ -0,0 +1,279 @@
+
+=== Built-In Feature Filters
+
+There are a few feature filters that come with the `azure-spring-cloud-feature-management` package. These feature filters are not added automatically, but can be setup in an `@Configuration` for use.
+
+==== AlwaysOnFilter
+
+This filter always returns true.
+
+==== PercentageFilter
+
+This filter provides the capability to enable a feature based on a set percentage.
+
+[source,yaml,indent=0]
+----
+feature-management:
+  feature-v:
+    enabled-for:
+      -
+        name: PercentageFilter
+        parameters:
+          percentage-filter-setting: 50
+----
+
+==== TimeWindowFilter
+
+This filter provides the capability to enable a feature based on a time window. If only `time-window-filter-setting-end` is specified, the feature will be considered on until that time. If only start is specified, the feature will be considered on at all points after that time. If both are specified the feature will be considered valid between the two times.
+
+[source,yaml,indent=0]
+----
+feature-management:
+  feature-v:
+    enabled-for:
+      -
+       name: TimeWindowFilter
+        parameters:
+          time-window-filter-setting-start: "Wed, 01 May 2019 13:59:59 GMT",
+          time-window-filter-setting-end: "Mon, 01 July 2019 00:00:00 GMT"
+----
+
+==== TargetingFilter
+
+This filter provides the capability to enable a feature for a target audience. An in-depth explanation of targeting is explained in the targeting section below. The filter parameters include an audience object that describes users, groups, and a default percentage of the user base that should have access to the feature. Each group object that is listed in the target audience must also specify what percentage of the group's members should have access. If a user is specified in the users section directly, or if the user is in the included percentage of any of the group rollouts, or if the user falls into the default rollout percentage, then that user will have the feature enabled.
+
+[source,yaml,indent=0]
+----
+feature-management: 
+  target:
+    enabled-for:
+      -
+        name: targetingFilter
+        parameters:
+          users:
+            - Jeff
+            - Alicia
+          groups:
+            -
+              name: Ring0
+              rolloutPercentage: 100
+            -
+              name: Ring1
+              rolloutPercentage: 100
+          defaultRolloutPercentage: 50
+----
+
+=== Custom Feature Filters
+
+Creating a feature filter provides a way to enable features based on criteria that you define. To implement a feature filter, the `FeatureFilter` interface must be implemented. `FeatureFilter` has a single method `evaluate`. When a feature specifies that it can be enabled with a feature filter, the `evaluate` method is called. If `evaluate` returns `true` it means the feature should be enabled. If `false` it will continue evaluating the Feature's filters until one returns true. If all return `false` then the feature is off.
```

Comment:
This does not include the word custom? It sounds like you are speaking about generic feature flags - "Creating a custom feature filter"
To create a custom feature filter, the `FeatureFilter` interface must be implemented.